Rakefile

```diff
@@ -47,7 +46,7 @@ task package: :gemspec
 
 RDoc::Task.new do |rd|
   rd.main = 'README.md'
-  rd.rdoc_files.include('README.md', 'lib/**/*.rb')
+  rd.rdoc_files.include('README.md', 'lib/*.rb')
```

Comment:
This will limit you to a lib structure without folders, is it intentional?

Reply:
It isn't. I went with the bare minimum necessary (i.e. the defaults that RSpec provided) to get the tests up and running.